src/angularComponent/ngcOmniboxSuggestionItemDirective.js

```diff
@@ -4,13 +4,19 @@ export default function ngcOmniboxSuggestionsDirective() {
   return {
     restrict: 'AE',
     require: ['^^ngcOmnibox', '^^ngcOmniboxSuggestions'],
-    scope: true,
+    scope: false,
```

Comment:
Uh oh, this is dangerous! Every item now is going to attach stuff to the scope that's shared between all the items, over-writing each other!

I think you spoke to me about "isolate scope" working, but `false` is not isolate scope, it's the total opposite!

`scope: false` = "don't make a new scope at all, use the same exact scope as parent"
`scope: true` = "make a new scope that prototypically inherits from the parent scope" 
                           (i.e. `newScope = Object.create(parentScope)`

`scope: {}` = isolate scope, only the bindings get passed in